packages/components/src/components/Button/Button.tsx

```diff
@@ -49,27 +49,35 @@ export type ButtonProps = {
      * @default false
      */
     disabled?: boolean;
+    /**
+     * @default 'default'
+     */
+    aspect?: ButtonAspect;
     /**
      * @default true
      */
     hugWidth?: boolean;
+    /**
+     * @deprecated please pass the Icon as a child and use the `aspect` prop
+     */
     icon?: ReactElement;
```

Comment:
as it is the new component, can be removed, wdyt?

Reply:
true, removed